conf/app.routes

```diff
@@ -7,9 +7,8 @@ GET        /cases/:reference                              @uk.gov.hmrc.bindingta
 POST       /cases                                         @uk.gov.hmrc.bindingtariffclassification.controllers.CaseController.create
 PUT        /cases/:reference                              @uk.gov.hmrc.bindingtariffclassification.controllers.CaseController.update(reference: String)
 PUT        /cases/:reference/status                       @uk.gov.hmrc.bindingtariffclassification.controllers.CaseController.updateStatus(reference: String)
-
-# DIT-290
-#PUT       /cases/:reference/decision                     @uk.gov.hmrc.bindingtariffclassification.controllers.CaseController.updateDecision(reference: String)
+DELETE     /cases                                         @uk.gov.hmrc.bindingtariffclassification.controllers.CaseController.deleteAll()
```

Comment:
Good to discus now. Could be good to have a different routes for QA and PROD this will avoid all the code above to enable/disable some functionalities in prod. 

A similar idea could be this 
https://stackoverflow.com/questions/21024292/does-play-support-separate-routes-files-per-environment-dev-uat-prod

But the main idea is to have an app.routes with the common ones and a QA.routes wich will inherit app.routes and will add the delete one for example

Reply:
I believe we cannot do this. 
All MDTP environments use the Prod Play environment.

That's why I introduced the boolean config.
This is the practice in HMRC Digital

Reply:
Good point isn't there a `prod.routes` file?

Reply:
not right now, we only have app.routes

Reply:
Could we check this ?
 https://github.com/hmrc/tracking-stub/blob/master/app/config/ApplicationLoader.scala

Reply:
`prod.routes` will be used in all MDTP environments anyway